src/dynarec/rv64/dynarec_rv64_emit_math.c

```diff
@@ -399,3 +399,46 @@ void emit_sbb8c(dynarec_rv64_t* dyn, int ninst, int s1, int c, int s3, int s4, i
     MOV32w(s6, c&0xff);
     emit_sbb8(dyn, ninst, s1, s6, s3, s4, s5);
 }
+
+// emit SBB32 instruction, from s1, s2, store result in s1 using s3 and s4 as scratch
+void emit_sbb32(dynarec_rv64_t* dyn, int ninst, rex_t rex, int s1, int s2, int s3, int s4, int s5)
+{
+    IFX(X_PEND) {
+        SDxw(s1, xEmu, offsetof(x64emu_t, op1));
+        SDxw(s2, xEmu, offsetof(x64emu_t, op2));
+        SET_DF(s3, rex.w?d_sbb64:d_sbb32);
+    } else IFX(X_ALL) {
+        SET_DFNONE();
+    }
+
+    IFX(X_AF | X_CF | X_OF) {
+        // for later flag calculation
+        NOT(s5, s1);
+    }
+
+    SUBxw(s1, s1, s2);
+    ANDI(s3, xFlags, 1 << F_CF);
+    BEQZ(s3, 8);
```

Comment:
`SUB(s1, s1, s3);` maybe instead of the `BEQZ`+`ADDI`?

Reply:
Yes, fixed.